array-manipulation-2/array-manipulation-2/src/chunk.js

```diff
@@ -0,0 +1,15 @@
+/* exported chunk */
+function chunk(array, size) {
+  var Arrays = [];
```

Comment:
you don't want to use `capital` letter for variables except in certain instances like defining classes.